cmd/mt-gateway/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/grafana/metrictank/logger"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"net/url"
+	"runtime"
+)
+
+var (
+	version       = "(none)"
+	showVersion   = flag.Bool("version", false, "print version string")
+	metrictankUrl = flag.String("metrictank-url", "http://localhost:6060", "metrictank address")
+	graphiteURL   = flag.String("graphite-url", "http://localhost:8080", "graphite-api address")
+	importerURL   = flag.String("importer-url", "", "mt-whisper-importer-writer address")
+	addr          = flag.String("addr", ":80", "http service address")
```

Comment:
per @woodsaj we should have a default org-id that is passed along if one hasn't been set on the incoming request.

Reply:
added in 967777966685ba71565db25bf08c177cd8c48fc8 (and refactored in subsequent commits)